scripts/src/osd/sdl_cfg.lua

```diff
@@ -153,6 +151,10 @@ elseif _OPTIONS["targetos"]=="freebsd" then
 		-- /usr/local/include is not considered a system include director on FreeBSD.  GL.h resides there and throws warnings
 		"-isystem /usr/local/include",
 	}
+elseif _OPTIONS["targetos"]=="android" then
+	includedirs {
+		_OPTIONS["ANDROID_SDL_HOME"] .. "/inc",
+	}
```

Comment:
No, in case of ndk-build is being used, only "libs" and "obj" directories are created, and subdir for each architecture in them, containing .so and .o, .d files. Nothing else of help unfortunately.

Reply:
Managed to solve this by using CMake for building Android library as also mentioned in one of scripts inside SDL2 distribution itself